gokart/tree/task_info.py

```diff
@@ -107,3 +118,40 @@ def make_tree_info_string(task: TaskOnKart, details: bool = False, abbr: bool =
     task_info = _make_task_info_tree(task, ignore_task_names=ignore_task_names)
     result = _make_tree_info(task_info=task_info, indent='', last=True, details=details, abbr=abbr, visited_tasks=set())
     return result
+
+
+def _make_tree_info_table_list(task_info: TaskInfo, visited_tasks: Set[str]):
+    task_id = task_info.get_task_id()
+    if task_id in visited_tasks:
+        return []
+    visited_tasks.add(task_id)
+
+    result = [task_info.task_info_dict()]
+
+    children = task_info.children_task_infos
+    for child in children:
+        result += _make_tree_info_table_list(task_info=child, visited_tasks=visited_tasks)
```

Comment:
[nits] you can use list comprehension

[super nits] you can also use list comprehension and flatten by `itertools.chain.from_iterable`.

```suggestion
    result += list(itertools.chain.from_iterable(_make_tree_info_table_list(task_info=child, visited_tasks=visited_tasks) for child in children))
```